hal/src/nRF52840/hal_platform_nrf52840_config.h

```diff
@@ -51,3 +51,5 @@
 #define HAL_PLATFORM_IFAPI (1)
 
 #define HAL_PLATFORM_ETHERNET (1)
+
+#define HAL_PLATFORM_BUTTON_DEBOUNCE_IN_SYSTICK (1)
```

Comment:
Please add the default to `hal_platform.h` as well.

Reply:
Added